HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -274,7 +275,8 @@ class HyprMXAdapter : PartnerAdapter {
             } successfully stored."
         )
 
-        if (prefsWriteSucceeded) {
+        // The HyprMX SDK throws an exception when the setConsentStatus API is used before the SDK has initialized.
+        if (prefsWriteSucceeded && HyprMX.getInitializationState() == HyprMXState.INITIALIZATION_COMPLETE) {
```

Comment:
we might want to log something if failed.
either that, or keep it around. if init happens after, set the consent status after the init succeeds.
See ChartboostMediationInternal for some logic around that.